dataactcore/scripts/setupErrorDB.py

```diff
@@ -1,38 +1,29 @@
+from dataactbroker.app import createApp
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models import lookups
 from dataactcore.models.errorModels import FileStatus, ErrorType
-from dataactcore.interfaces.db import databaseSession
 
 
 def setupErrorDB():
     """Create error tables from model metadata."""
-    with databaseSession() as sess:
+
+    with createApp().app_context():
+        sess = GlobalDB.db().session
         insertCodes(sess)
         sess.commit()
 
+
 def insertCodes(sess):
     """Insert static data."""
 
     # insert file status types
-
-    statusList = [(1, 'complete', 'File has been processed'),
-        (2, 'header_error', 'The file has errors in the header row'),
-        (3, 'unknown_error', 'An unknown error has occurred with this file'),
-        (4, 'single_row_error', 'CSV file must have a header row and at least one record'),
-        (5, 'job_error', 'Error occurred in job manager'),
-        (6, 'incomplete', 'File has not yet been validated')]
-    for s in statusList:
-        status = FileStatus(file_status_id=s[0], name=s[1], description=s[2])
+    for s in lookups.FILE_STATUS:
+        status = FileStatus(file_status_id=s.id, name=s.name, description=s.desc)
```

Comment:
I'm not sure if this would be more readable, but I wanted to throw it out in case you liked it:
```python
for status_attrs in lookups.FILE_STATUS:
    status = FileStatus(*status_attrs)
```